Python/Source/Range.cpp

```diff
@@ -0,0 +1,13 @@
+#include "Range.hpp"
+
+using namespace pybind11;
+using namespace Rover;
+
+void Rover::export_range(module& module) {
+  class_<Details::PythonRange>(module, "Range")
+    .def(init<object, object>())
+    .def(init<object, object, object>())
```

Comment:
Similarly you need bindings for ```Interval``` enum and to make use of it in the ```Range```'s constructor.